File: player/src/main/java/ru/testsimpleapps/coloraudioplayer/ui/dialogs/AboutDialog.java
package ru.testsimpleapps.coloraudioplayer.ui.dialogs;

import android.content.Context;
import android.os.Bundle;

import ru.testsimpleapps.coloraudioplayer.R;

public class AboutDialog
        extends BaseDialog {

    private final Context context;

    public AboutDialog(Context context) {
        super(context);
        this.context = context;
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.dialog_about);
    }

}

File: player/src/main/java/ru/testsimpleapps/coloraudioplayer/ui/dialogs/AboutDialog.kt
package ru.testsimpleapps.coloraudioplayer.ui.dialogs

import android.content.Context
import android.os.Bundle

import ru.testsimpleapps.coloraudioplayer.R

class AboutDialog(context: Context) : BaseDialog(context) {

    override fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.dialog_about)
    }

}
